yt_dlp/extractor/litv.py

```diff
@@ -68,24 +60,33 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
+        if '<meta http-equiv="refresh" content="1;url=https://www.litv.tv/"' in webpage:
+            raise ExtractorError('No such content', expected=True)
+
         program_info = self._parse_json(self._search_regex(
             r'var\s+programInfo\s*=\s*([^;]+)', webpage, 'VOD data', default='{}'),
             video_id)
 
-        season_list = list(program_info.get('seasonList', {}).values())
-        playlist_id = traverse_obj(season_list, 0, 'contentId')
-        if self._yes_playlist(playlist_id, video_id, smuggled_data):
-            return self._extract_playlist(season_list[0], video_id, program_info)
-
-        # In browsers `getMainUrl` request is always issued. Usually this
+        # In browsers `getProgramInfo` request is always issued. Usually this
         # endpoint gives the same result as the data embedded in the webpage.
-        # If georestricted, there are no embedded data, so an extra request is
-        # necessary to get the error code
+        # If, for some reason, there are no embedded data, we do an extra request.
         if 'assetId' not in program_info:
             program_info = self._download_json(
                 'https://www.litv.tv/vod/ajax/getProgramInfo', video_id,
                 query={'contentId': video_id},
-                headers={'Accept': 'application/json'})
+                headers={'Content-Type': 'application/json',
```

Comment:
why send a `Content-Type` header if this is not a POST request?

Reply:
Well, it looks like this header is not necessary for the extractor to work (although it does get sent by browsers to litv.tv on this request). I’ll remove it.

The 405 error did not come from this request. It came from the request I fixed in  fd49568d9d0074dd3e44aa6d4af919ddc7808d52, the endpoint simply got changed from `/vod/getMainUrl` to `/vod/ajax/getMainUrlNoAuth`.